### ISSUE TITLE
Papa's Freeeria Deluxe does not boot

### ISSUE THREAD

**Issue by Fancy2209** [ruffle-rs/ruffle#15818]
### Describe the bug

The game fails to boot due to flash.display::AspectRatio beeing missing
```
2024-03-30T23:44:35.636041Z ERROR ruffle_core::display_object::movie_clip: Got "ReferenceError: Error #1065: Variable flash.display::StageAspectRatio is not defined.
        at iceGame.core::GameModule/initGame()
        at Main/startup()
        at Main()" when constructing AVM2 side of movie clip of type Main
```
To obtain this error I had to modify the Try Catch on the Main script by leaving only the code from the Try part.
![imagem](https://github.com/ruffle-rs/ruffle/assets/64917206/33760763-3b96-4797-96e2-3bbf065020f2)

### Expected behavior

Game Would boot normally
![imagem](https://github.com/ruffle-rs/ruffle/assets/64917206/56b9a6e1-f44e-4108-a805-2738039663f1)

### Content Location

https://store.steampowered.com/app/2291760/Papas_Freezeria_Deluxe/
swf is located at `Papa's Freezeria Deluxe\Data\assets\Main.assets` (.assets are renamed .swf)

### Affected platform

Desktop app

### Operating system

Windows 11

**Comment by oadq** [comment]
@Fancy2209 
Have you set the player runtime to Adobe AIR? This class seems to be implemented.

**Comment by Fancy2209** [comment]
> @Fancy2209 Have you set the player runtime to Adobe AIR? This class seems to be implemented.

I'm aware, I was the one who stubbed it after all, it's still missing stuff, I will update the issue once 3 AIR related PRs I'm waiting on get merged

**Comment by qtqgyt** [comment]
Also encountering this issue on the latest nightly. Any plans for implementing Adobe AIR support?

**Comment by Fancy2209** [comment]
> Also encountering this issue on the latest nightly. Any plans for implementing Adobe AIR support?

We have air support
There was a PR who had some things who let it progress but it went stale since the dev got busy

**Comment by Fancy2209** [comment]
> Also encountering this issue on the latest nightly. Any plans for implementing Adobe AIR support?

Wait but StageAspectRatio was added months ago
Are you sure you aren't thinking of something else?

**Comment by qtqgyt** [comment]
To be more accurate, it crashes shortly after showing the this game can't be run in flash player message. Ruffle just exits with no logs.

**Comment by Fancy2209** [comment]
That is a different error, post ruffle's log file

**Comment by qtqgyt** [comment]
> That is a different error, post ruffle's log file

Sorry, I wasn't sure if it belonged here or not. I will probably crate a new issue. Thanks!
